recipe.yaml

```diff
@@ -25,6 +20,38 @@
       - python3-libpagure
       - python3-fedmsg
       state: present
+  - name: Install dependencies for syncer
+    dnf:
+      name:
+      - rpm-build
+      state: present
+  - name: Install latest twine for sake of check command
+    pip:
+      name:
+      - twine  # we need newest twine, b/c of the check command
+      - readme_renderer[md]
+      state: latest
+  - name: "Set up git: user.email"
+    command: git config --global user.email "source-git@example.org"
+  - name: "Set up git: user.name"
+    command: git config --global user.name "Source Git"
+  - name: Create .ssh
+    file:
+      state: directory
+      path: /root/.ssh
+      mode: 0700
+  - name: Copy keys in there
+    copy:
+      src: '{{ item.s }}'
+      dest: /root/.ssh/{{ item.d }}
+      mode: 0400
+    with_items:
+    - {s: "bigger-secret", d: "id_rsa.pub"}
+    - {s: "biggest-secret", d: "id_rsa"}
```

Comment:
What about some better name? But it is not so important.

Reply:
Should be fixed now; I also added a note to readme about the need for those files